mongo/integration/unified/unified_spec_test.go

```diff
@@ -79,6 +80,7 @@ type TestFile struct {
 	CreateEntities    []map[string]*EntityOptions `bson:"createEntities"`
 	InitialData       []*CollectionData           `bson:"initialData"`
 	TestCases         []*TestCase                 `bson:"tests"`
+	YamlAnchors       map[string]interface{}      `bson:"_yamlAnchors"`
```

Comment:
This field can be ignored by test runners and our BSON library doesn't error if there's a field in the BSON/JSON that's not represented in the target struct, so we should be able to remove this.

Reply:
Sounds good, had to rebase with [1235](https://jira.mongodb.org/browse/GODRIVER-1805) to get that undefined field to skip correctly.